randovania/games/dread/json_data/Dairon.txt

```diff
@@ -2294,105 +2304,58 @@ Extra - asset_id: collision_camera_025
   > Beside the Blob
       Trivial
 
-> Tile Group (SPEEDBOOST) 1; Heals? False
-  * Layers: default
-  * Configurable Node
-  * Extra - actor_name: breakabletilegroup_015
-  * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
-  * Extra - actor_layer: breakables
-  * Extra - tile_types: ('SPEEDBOOST',)
-  > Bottom-Left Corner
-      Trivial
-
-> Tile Group (SPEEDBOOST) 2; Heals? False
+> Tile Group (SCREWATTACK); Heals? False
   * Layers: default
   * Configurable Node
-  * Extra - actor_name: breakabletilegroup_016
+  * Extra - actor_name: breakabletilegroup_030
   * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
   * Extra - actor_layer: breakables
-  * Extra - tile_types: ('SPEEDBOOST',)
-  > Tile Group (SCREWATTACK)
-      All of the following:
-          Space Jump or Speed Booster or Walljump (Beginner) or Simple IBJ
-          Any of the following:
-              Gravity Suit
-              Heat/Cold Runs (Intermediate) and Cold Damage ≥ 100
+  * Extra - tile_types: ('SCREWATTACK',)
   > Tile Group (WEIGHT)
       All of the following:
           Morph Ball
           Any of the following:
-              Gravity Suit
-              Heat/Cold Runs (Intermediate) and Cold Damage ≥ 50
-  > Bottom-Left Corner
-      Trivial
-
-> Tile Group (SPEEDBOOST) 3; Heals? False
-  * Layers: default
-  * Configurable Node
-  * Extra - actor_name: breakabletilegroup_017
-  * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
-  * Extra - actor_layer: breakables
-  * Extra - tile_types: ('SPEEDBOOST',)
-  > Tile Group (SCREWATTACK)
-      Trivial
+              Gravity Suit or Before Dairon - powergenerator_002
+              Heat/Cold Runs (Intermediate) and Cold Damage ≥ 70
   > Freezer Center
-      Gravity Suit
-
-> Tile Group (SPEEDBOOST) 4; Heals? False
-  * Layers: default
-  * Configurable Node
-  * Extra - actor_name: breakabletilegroup_018
-  * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
-  * Extra - actor_layer: breakables
-  * Extra - tile_types: ('SPEEDBOOST',)
-  > Door to Energy Recharge Station West (doorpowerpower_028)
       Any of the following:
-          Gravity Suit
-          Heat/Cold Runs (Intermediate) and Cold Damage ≥ 20
-  > Door to Energy Recharge Station West (doorpowerpower_030)
-      All of the following:
-          Morph Ball and After Dairon - db_dif_b1_001
-          Any of the following:
-              Gravity Suit
-              Heat/Cold Runs (Intermediate) and Cold Damage ≥ 100
-
-> Tile Group (SCREWATTACK); Heals? False
```

Comment:
"Tile Group (SCREWATTACK)" -> "Bottom-Left Corner" should be an available connection that requires speedbooster.

As it is, there is only one path to bottom-left corner, which is "Tile Group (SCREWATTACK)" -> "Tile Group (WEIGHT)" -> "Bottom-Left Corner", which requires morph ball, when that isn't necessarily required to get to the bottom left corner.

Reply:
it is required to get the item though, which is the only reason to go down there, but i probably should do that

Reply:
ah you do have a point. but yeah I guess we can keep the extra path